Logger/SerilogConfiguration.cs

```diff
@@ -0,0 +1,29 @@
+using Serilog;
+
+namespace QuickEdit.Logger;
+
+public class SerilogConfiguration
+{
+
+	public static void ConfigureLogger()
+	{
+		var intermediateOutputPath =
+			Path.GetFullPath(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"..\..\..", @"obj\Debug\net8.0\Logs"));
+
+		Directory.CreateDirectory(intermediateOutputPath);
+
+		var logPath = Path.Combine(intermediateOutputPath, $"consoleapplog-{DateTime.UtcNow:yyyyMMddHHmmss}.txt");
+
+		var loggerConfig = new LoggerConfiguration()
+			.Enrich.WithMachineName()
```

Comment:
I don't think this enrichment is used at all here. To use it, I think a custom `outputTemplate` should be used. However, I don't think this feature is necessary in this case, so it could just be removed along with the `Serilog.Enrichers.Environment` NuGet package.

```suggestion
```

If you think it should stay, let me know